### PR TITLE
Add varnish 6 support

### PR DESCRIPTION
**Pull request by Cactusbone**
This is mainly code taken from https://github.com/collectd/collectd/pull/3360, but without losing previous varnish versions.

ChangeLog: varnish plugin: Add varnish 6 support.
ChangeLog: varnish plugin: Add counters for vmod-goto.

### CODE REVIEW

File: src/varnish.c [hunk 1]
```diff
@@ -478,6 +496,27 @@ static int varnish_monitor(void *priv,
...
+                                   val);
+    else if (strcmp(name, "sess_fail_eintr") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failed_eintr", val);
+    else if (strcmp(name, "sess_fail_emfile") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failed_emfile", val);
+    else if (strcmp(name, "sess_fail_ebadf") == 0)
+      return varnish_submit_derive(conf->instance, "session",
+                                   "total_operations", "failedi_ebadf", val);
```
Comment (octo) [inline src/varnish.c:512]: Typo, surplus "i"

Comment (Cactusbone) [reply src/varnish.c:512]: good catch ! Should I squash the fix ?

Comment (octo) [reply src/varnish.c]: Looks like you already did this, thanks!

**Comment by octo** [comment]
Thanks Charly!

**Pull request merged**